server/config/store_config.go

```diff
@@ -118,6 +119,32 @@ func (c *StoreConfig) GetRegionMaxKeys() uint64 {
 	return uint64(c.Coprocessor.RegionMaxKeys)
 }
 
+// CheckRegionSize return error if the smallest region's size is less than mergeSize
+func (c *StoreConfig) CheckRegionSize(size, mergeSize uint64) error {
+	if size < c.GetRegionMaxSize() {
+		return nil
+	}
+
+	if smallSize := size % c.GetRegionSplitSize(); smallSize <= mergeSize && smallSize != 0 {
+		log.Info("region size is too small", zap.Uint64("size", size), zap.Uint64("mergeSize", mergeSize), zap.Uint64("smallSize", smallSize))
+		return errs.ErrCheckerMergeAgain.FastGenByArgs("the smallest region of the split regions is less than max-merge-region-size, " +
+			"it will be merged again")
+	}
+	return nil
+}
+
+// CheckRegionKeys return error if the smallest region's keys is less than mergeKeys
+func (c *StoreConfig) CheckRegionKeys(keys, mergeKeys uint64) error {
+	if keys < c.GetRegionMaxKeys() {
```

Comment:
better to add some comments on `RegionMaxKeys`,  `Region SplitKeys`。

Reply:
fix